vm/src/vm/vm_object.rs

```diff
@@ -53,6 +53,11 @@ impl VirtualMachine {
             error(&s);
             panic!("{}; exception backtrace above", msg)
         }
+        #[cfg(all(target_arch = "wasm32", not(feature = "js"), not(target_os = "wasi")))]
```

Comment:
Wouldn't it be useful to still see information from the exception? Can't you do something like:

```rust
let err_string: String = exc.to_pyobject(vm).repr(vm).unwrap().to_string();
```

Or does the `msg` have everything we need?

Reply:
It will be useful